README.md

```diff
@@ -85,6 +85,14 @@ document. Lifecycle stages are defined in the
 
 - [1]: See [stability guarantees](./specification/versioning.md) for details.
 
+## Experimental
+
+Everything in the specification starts as `Experimental`.
+
+`Experimental` sections SHOULD NOT be expected to be feature-complete.
+In some cases, the experiment MAY be discarded and removed entirely.
```

Comment:
Feels like there is a word missing after `experimental` on this line. Maybe it should read `experimental sections`?

Reply:
This is `experiment` a  noun. Not an adjective.
I understand an `experiment` as experimental section/feature 😉

Reply:
oh, misread it